clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/provider/agent/TargetHealthCachingAgent.java

```diff
@@ -114,7 +115,15 @@ public TargetHealthCachingAgent(
     Collection<String> targetGroupKeys =
         awsProviderCache.filterIdentifiers(TARGET_GROUPS.getNs(), searchKey);
 
-    List<EcsTargetGroup> tgList = cacheClient.find(targetGroupKeys);
+    if (targetGroupKeys.size() > 0) {
+      log.debug("Found " + targetGroupKeys.size() + " target group keys in " + getAgentType());
```

Comment:
Would it be useful to also log the number of results for this account after filtering below? Or is that logged elsewhere already

Reply:
the number of target groups returned by this func (after filtering by account and deduping) is logged on line 75, prior to calling `describeTargetHealth`

Reply:
No that works